### ISSUE TITLE
Drop @nextcloud/text package build, dependencies and code bits

### ISSUE THREAD

**Issue by juliusknorr** [nextcloud/text#5051]
As we moved to the text API exposed as a global function we should probably clean up parts related to the `@nextcloud/text` package at some point.

cc @mejo-

**Comment by mejo-** [comment]
Indeed, we should drop this at one point. I'd say let's wait 1-2 months though, in case we still have to release a new version with a critical fix. For the record, Collectives uses the editor API starting with server releases 26.0.8 and 27.1.3.

**Comment by juliusknorr** [comment]
I don't see any leftovers anymore on main. Let's close this.
